src/System.Management.Automation/CoreCLR/CorePsPlatform.cs

```diff
@@ -195,7 +197,7 @@ internal static bool IsInbox
         /// Some common environment variables used in PS have different
         /// names in different OS platforms
         /// </summary>
-        internal static class CommonEnvVariableNames
+        public static class CommonEnvVariableNames
```

Comment:
It's just a constant - makes no sense.
We would discuss this in the topic about creating platform unified scripts to get common solution and best practices. But we have not yet created such a discussion.